pulsar-client/pom.xml

```diff
@@ -136,6 +136,12 @@
       <version>${project.parent.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.10.1</version>
```

Comment:
Please define the dependency and version in top level Pom.xml in the dependenciesManagement section, to ensure we a have a single version

Reply:
all right,I wiil do it

Reply:
@merlimat @congbobo184 
sorry, I PR #3856 add avro LogicalTypes.date()/timeMillis()/timestampMillis()

Reply:
@ambition119 #3900 is a bug with avro, I only repair it, it can't influence each other.

Reply:
@merlimat I think avro schema should use joda, #3856 should fix the pulsar schema. It shouldn't to change the avro schema.

Reply:
> @merlimat I think avro schema should use joda, #3856 should fix the pulsar schema. It shouldn't to change the avro schema.

ok, I fix this content.

Reply:
just FieldSchemaBuilderImpl.java

Reply:
just FieldSchemaBuilderImpl.java